README.rst

```diff
@@ -27,8 +27,22 @@ BigchainDB Python Driver
 Features
 --------
 
-* Minimal support for `CREATE`, `TRANSFER`, and `GET` operations on the
-  `/transactions` endpoint.
+* Support for preparing, fulfilling, and sending transactions to a BigchainDB
+  node.
+* Retrieval of transactions by id.
+* Getting status of a transaction by id.
+
+
+Compatibility Matrix
+--------------------
+
++-----------------------+-----------------------+
+| **BigchainDB Server** | **BigchainDB Driver** |
++=======================+=======================+
+| ``0.7.x``             | ``0.0.3``             |
++-----------------------+-----------------------+
+| ``master``            | ``master``            |
```

Comment:
Yes, it would be worthwhile to note that the two master branches sometimes get out of sync.